src/Apps/Order/Components/SaveAndContinueButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from "react"
+import { Button, Spacer } from "@artsy/palette"
+import { Media } from "Utils/Responsive"
+
+export const SaveAndContinueButton: FC<{
+  onClick?: () => void
+  media?: { [key: string]: string }
+  loading?: boolean
+  disabled?: boolean
+}> = ({ onClick, media, loading = false, disabled = false }) => (
```

Comment:
As code is always returned to (and never truly complete!) we can be sure that in the future some dev will open this file and want to either expand it with a new feature, or console.log a pro, or any number of other things. This implicit return will make that much harder, requiring the dev to rewrite the format to have a (normal) explicit return. 

The convention is: if its a single liner, implicit returns are totally cool; think `const z = (x) => x * y`. 

Anything more than one line means there's some sufficient complexity therein and it will be a pain for someone else to have to unwind it later. 

There's a note about this in here: https://github.com/artsy/force/blob/main/docs/best_practices.md#avoid-implicit-returns-on-react-components

Reply:
makes sense, thanks for the heads up 👍